api/v1/views/states.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+"""
+Handles all default RestFul API actions for State class
+"""
+from flask import Flask, jsonify, Blueprint, make_response, render_template
+from api.v1.views import app_views
+from models import state, storage
+
+
+@app_views.route('/states', methods=['GET'], strict_slashes=False)
+def get_States():
+    """get all states"""
+    all_st = [st.to_json() for st in storage.all("State").values()]
+    return jsonify(all_st)
```

Comment:
change to:    return jsonify([st.to_json() for st in storage.all("State").values()]) to make one line.